src/services/services.ts

```diff
@@ -4677,11 +4677,13 @@ namespace ts {
                 let definition: Declaration;
 
                 forEach(signatureDeclarations, d => {
-                    if ((selectConstructors && d.kind === SyntaxKind.Constructor) ||

```

Comment:
If I checked types in the main loop,  following tests were failed.
And I think shouldn't change these tests.

- [goToTypeDefinition2.ts](https://github.com/Microsoft/TypeScript/blob/master/tests/cases/fourslash/goToTypeDefinition2.ts)
- [goToTypeAliases.ts](https://github.com/Microsoft/TypeScript/blob/master/tests/cases/fourslash/goToTypeDefinitionAliases.ts)
- [goToTypeDefinitionUnionType.ts](https://github.com/Microsoft/TypeScript/blob/master/tests/cases/fourslash/goToTypeDefinitionUnionType.ts)